charts/timescaledb-multinode/templates/statefulset-timescaledb-accessnode.yaml

```diff
@@ -26,6 +26,9 @@ spec:
         app: {{ template "timescaledb.fullname" . }}
         release: {{ .Release.Name }}
         timescaleNodeType: access
+        {{ with .Values.accessNode.extraPodLabels -}}
+{{ toYaml . }}
```

Comment:
indentation
```
      {{- with .Values.accessNode.extraPodLabels }}
      {{ toYaml . | nindent 6/8}}
      {{- end }}
```

Reply:
tested and renders correctly:

`helm template timescaledb -f values.yaml -s templates/statefulset-timescaledb-accessnode.yaml .  
---
# Source: timescaledb-multinode/templates/statefulset-timescaledb-accessnode.yaml
# This file and its contents are licensed under the Apache License 2.0.
# Please see the included NOTICE for copyright information and LICENSE for a copy of the license.

apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: timescaledb-access
  labels:
    app: timescaledb
    chart: timescaledb-multinode-0.9.0
    release: timescaledb
    heritage: Helm
spec:
  serviceName: timescaledb-access
  replicas: 1
  podManagementPolicy: Parallel
  selector:
    matchLabels:
      app: timescaledb
      release: timescaledb
      timescaleNodeType: access
  template:
    metadata:
      name: timescaledb-access
      labels:
        app: timescaledb
        release: timescaledb
        timescaleNodeType: access
        app.kubernetes.io/part-of: globalcatalog
`

Reply:
```shell
(us-east-1.playground):/tmp/timescaledb-kubernetes/charts/timescaledb-multinode$ git diff
diff --git a/charts/timescaledb-multinode/templates/statefulset-timescaledb-accessnode.yaml b/charts/timescaledb-multinode/templates/statefulset-timescaledb-accessnode.yaml
index 55ab134..467c6fa 100644
--- a/charts/timescaledb-multinode/templates/statefulset-timescaledb-accessnode.yaml
+++ b/charts/timescaledb-multinode/templates/statefulset-timescaledb-accessnode.yaml
@@ -26,8 +26,8 @@ spec:
         app: {{ template "timescaledb.fullname" . }}
         release: {{ .Release.Name }}
         timescaleNodeType: access
-        {{ with .Values.accessNode.extraPodLabels -}}
-        {{ toYaml . }}
+        {{- with .Values.accessNode.extraPodLabels }}
+        {{- toYaml . | nindent 8 }}
         {{- end }}
     spec:
       serviceAccountName: {{ template "timescaledb.serviceAccountName" . }}
diff --git a/charts/timescaledb-multinode/values.yaml b/charts/timescaledb-multinode/values.yaml
index f264fb6..e269647 100644
--- a/charts/timescaledb-multinode/values.yaml
+++ b/charts/timescaledb-multinode/values.yaml
@@ -35,7 +35,9 @@ env:
     value: /var/lib/postgresql/pgdata

 accessNode:
-  extraPodLabels: {}
+  extraPodLabels:
+    test: one
+    another: test
   #enable external access using LoadBalancer
   service:
     type: LoadBalancer
(us-east-1.playground):/tmp/timescaledb-kubernetes/charts/timescaledb-multinode$ helm template . | grep test -A 2 -B 2
        release: RELEASE-NAME
        timescaleNodeType: access
        another: test
        test: one
    spec:
      serviceAccountName: RELEASE-NAME-timescaledb
(us-east-1.playground):/tmp/timescaledb-kubernetes/charts/timescaledb-multinode$ git checkout templates/statefulset-timescaledb-accessnode.yaml
Updated 1 path from the index
(us-east-1.playground):/tmp/timescaledb-kubernetes/charts/timescaledb-multinode$ git diff
diff --git a/charts/timescaledb-multinode/values.yaml b/charts/timescaledb-multinode/values.yaml
index f264fb6..e269647 100644
--- a/charts/timescaledb-multinode/values.yaml
+++ b/charts/timescaledb-multinode/values.yaml
@@ -35,7 +35,9 @@ env:
     value: /var/lib/postgresql/pgdata

 accessNode:
-  extraPodLabels: {}
+  extraPodLabels:
+    test: one
+    another: test
   #enable external access using LoadBalancer
   service:
     type: LoadBalancer
(us-east-1.playground):/tmp/timescaledb-kubernetes/charts/timescaledb-multinode$ helm template . | grep test -A 2 -B 2
Error: YAML parse error on timescaledb-multinode/templates/statefulset-timescaledb-accessnode.yaml: error converting YAML to JSON: yaml: line 31: mapping values are not allowed in this context

Use --debug flag to render out invalid YAML
```